src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/LocalDBTest/LocalDBTest.cs

```diff
@@ -26,15 +26,15 @@ private enum InfoType
         private static readonly string s_startLocalDbCommand = @$"/c SqlLocalDb start {DataTestUtility.LocalDbAppName}";
         private static readonly string s_localDbNamedPipeConnectionString = @$"server={GetLocalDbNamedPipe()}";
 
+#if !WINDOWS_UAP
         #region LocalDbTests
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // No Registry support on UAP
+
```

Comment:
Replacing this attribute with a directive will skip showing these tests in the report in unsupported TF. There should be a solution to detect it in a utility. [PalatformDetection](https://github.com/dotnet/runtime/blob/f7a7b2b8b12dac91aa86a31e59ef6c1028d7113e/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs) from runtime would be helpful.